### ISSUE TITLE
Measure scrubadub accuracy on real data

### ISSUE THREAD

**Issue by thomasbird** [LeapBeyond/scrubadub#72]
We can measure scrubadub accuracy with fake data (see #59), it would be good to find real data sources where this is also possible.

**Comment by TheBellman** [comment]
It would be useful to include that as a repeatable test

**Comment by thomasbird** [comment]
Yes agreed, we should do something similar to the fake data, where it is run in the CI process for speedy models (and offline for slower models):

https://travis-ci.org/github/LeapBeyond/scrubadub/jobs/741856233#L616

**Comment by thomasbird** [comment]
Thoughts:

- Wikipedia  list of people
- Addresses/phone numbers of take away places

**Comment by aCampello** [comment]
This might be useful for the addresses: https://www.kaggle.com/openaddresses/openaddresses-europe

**Comment by aCampello** [comment]
For the UK locale, for addresses and postcode, we can perhaps use the Companies House open database https://www.gov.uk/government/organisations/companies-house
